conf/modules.config

```diff
@@ -579,7 +579,7 @@ if (!params.skip_peak_annotation) {
             ext.args   = '-gid'
```

Comment:
The name in the previous line is not updated `BAM_PEAKS_CALL_QC_ANNOTATE_MACS2_HOMER:HOMER_ANNOTATEPEAKS`

Reply:
Good spot, did not save the change 😓